### ISSUE TITLE
Error 404 (Not Found)!!1

### ISSUE THREAD

**Issue by LouisCoates** [AngeloD2022/onhubauthhelper#10]
Hi There,

I manally downloaded the zip file for this code, and installed it as a chrome extention, but whenever I try and go to the "Consent Page" I get met with an "Error 404 (Not Found)!!1". The link to this page is:

https://accounts.google.com/o/oauth2/programmatic_auth?scope=https%3A%2F%2Fwww.google.com%2Faccounts%2FOAuthLogin&client_id=936475272427.apps.googleusercontent.com&device_name=google-wifi

Is this a known issue?

![image](https://user-images.githubusercontent.com/23196110/225896719-a1b03f99-0b30-4383-b11c-ed93f01f18ad.png)

**Comment by ywaf** [comment]
yeah same, only just started happening. worked fine like two weeks ago :/

**Comment by Peter-HVD** [comment]
Me too :-(

**Comment by Nattle** [comment]
+1

**Comment by AngeloD2022** [comment]
My bad, that was a misclick. But besides that, I don't currently have the time to comprehensively investigate this problem. I am preoccupied with other projects. This is worth revisiting though, and I will keep this on my bucket list.

**Comment by Nattle** [comment]
@AngeloD2022 Any other suggested work around? For now the integration is completely dead in the water for new users.

**Comment by AngeloD2022** [comment]
I have no knowledge of any other workaround that is not laborious and difficult. Even then, I do not know if the method I have in mind is still functional. I may do a writeup if it still works. If ever I do, it will be published on https://docs.angelod.com.
